### PR TITLE
crowbar.0.2

### PR DESCRIPTION
**Pull request by stedolan**
New release of crowbar (finally!)

### CODE REVIEW

**Comment by stedolan** [comment]
Build failures were uses of `List.init` (needs 4.06) and `Stdlib` (needs 4.08, I think?). I've bumped the required version to 4.08 rather than patching uses.

File: packages/crowbar/crowbar.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+license: "MIT"
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+run-test: [
+  [ "dune" "runtest" "-p" name "-j" jobs ]
+]
+depends: [
+  "ocaml" {>= "4.08"}
+  "jbuilder"
```
Comment (hannesm) [inline packages/crowbar/crowbar.0.2/opam:15]:
```suggestion
  "dune" {>= "1.1"}
```

File: packages/crowbar/crowbar.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "ocplib-endian"
+  "cmdliner"
+  "afl-persistent" {>= "1.1"}
+  "calendar" {with-test}
+  "xmldiff" {with-test}
+  "fpath" {with-test}
```
Comment (hannesm) [inline packages/crowbar/crowbar.0.2/opam:21]:
```suggestion
  "fpath" {with-test}
  "pprint" {with-test & < "20180528"}
```

**Comment by kit-ty-kate** [comment]
Note that tests can't be run anymore (at least as long as xmldiff is not available on OCaml >= 4.08):
```
$ opam depext -uivyj 72 crowbar.0.2
[ERROR] No solution for crowbar.0.2: The following dependencies couldn't be met:
          - crowbar -> xmldiff -> xmlm < 1.3.0 -> ocaml < 4.06.0
              base of this switch (use `--unlock-base' to force)
```

**Comment by XVilka** [comment]
And [xmldiff](https://opam.ocaml.org/packages/xmldiff/) seems abandoned: https://github.com/zoggy/xmldiff/ (last commit was in 2017). So the only solution would be make a fork of it. Last release was in 2014...

**Comment by stedolan** [comment]
@hannesm thanks, applied. (these were already fixed in crowbar's repo, but got lost in a freak copypaste accident for this repo)

@kit-ty-kate I've changed the opam run-test block to explicitly list the tests that run, I'll fix them upstream for the next release.

File: packages/crowbar/crowbar.0.2/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  [ "dune" "build" "-p" name "-j" jobs
+    "@examples/calendar/runtest"
+    "@examples/fpath/runtest"
+    "@examples/map/runtest"
+    "@examples/serializer/runtest"
+    "@examples/uunf/runtest"
+  ]
+]
+depends: [
+  "dune" {build & >= "1.1"}
```
Comment (kit-ty-kate) [inline packages/crowbar/crowbar.0.2/opam:20]: Could you remove the `build` flag here? Dune is not compatible with this flag and I can't edit your PR.

**Comment by kit-ty-kate** [comment]
Sorry for the delay. I'll make the change shortly after merging this. Thanks!

**Pull request merged**